contracts/test/Gateway.t.sol

```diff
@@ -81,6 +81,7 @@ contract GatewayTest is Test {
     uint128 public registerTokenFee = 0;
     uint128 public sendTokenFee = 1e10;
     uint128 public createTokenFee = 1e10;
+    uint128 public maxDestinationFee = 1e11;
```

Comment:
How do we deal with the difference in precision between ROC+KSM and DOT? Because this is 10 DOT I think, but 0.1 KSM and 0.1 ROC.